mirgecom/integrators.py

```diff
@@ -79,8 +59,25 @@ def lsrk4_step(state, t, dt, rhs):
     k = p * 0.
 
     for i in range(5):
-        k = _LSRK4_A[i]*k + dt*rhs(t + _LSRK4_C[i]*dt, p)
-        p = p + _LSRK4_B[i]*k
+        k = bt._LSRK4_A[i]*k + dt*rhs(t + bt._LSRK4_C[i]*dt, p)
+        p = p + bt._LSRK4_B[i]*k
+
+    return p
+
+
+def lsrk144_step(state, t, dt, rhs):
+    """
+    Take one step using the low storage 14-stage 4th order Runge-Kutta method.
+
+    LSRK coefficients are summarized in Table 3 of Niegemann, Diehl, and
+    Busch (2012): https://doi.org/10.1016/j.jcp.2011.09.003.
```

Comment:
Thanks a ton, Thomas!  Could you please add the reference to `doc/misc.rst` and reference it here?

Reply:
Will do!